.travis.yml

```diff
@@ -162,10 +156,9 @@ before_install:
 
 ### Note: Fix up SASL pkgconfig before build
 before_script:
-- ./.travis.fixsasl
```

Comment:
I think the leading `-` was needed here, as it is a YAML list (of operations to run)